plugins/sla-email/src/main/java/com/thinkbiganalytics/metadata/sla/SlaEmailService.java

```diff
@@ -51,12 +54,23 @@
      * @param body    the email body
      */
     public void sendMail(String to, String subject, String body) {
-        SimpleMailMessage message = new SimpleMailMessage();
-        message.setFrom(emailConfiguration.getFrom());
-        message.setTo(to);
-        message.setSubject(subject);
-        message.setText(body);
-        mailSender.send(message);
+
+        try {
+            if(testConnection()) {
+                MimeMessage message = mailSender.createMimeMessage();
+                String fromAddress = StringUtils.defaultIfBlank(emailConfiguration.getFrom(),emailConfiguration.getUsername());
+                message.setFrom(new InternetAddress(fromAddress));
+                message.setRecipients(Message.RecipientType.TO, InternetAddress.parse(to));
```

Comment:
Refactored message type to MIME as it can send to multiple recipients, allowing me to remove the extra code in EmailServiceLevelAgreementAction.java